docs/api-documentation/index.html

```diff
@@ -60,14 +60,14 @@ <h1 class="govuk-heading-m">Releases</h1>
   <ul class="govuk-list govuk-list--bullet">
     <li>
       <!-- UPDATE ME to current latest version -->
-      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.3/">
-        Version 3.3 - Released to Sandbox on 10/06/2021. Released to Production on 14/06/2021.
+      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.4/">
+        Version 3.4 - Released to Sandbox on 24/06/2021. Released to Production on 28/06/2021.
```

Comment:
Do we usually do something like this?

```suggestion
        Version 3.4 - Released to Sandbox on 24/06/2021. Due for release to Production on 28/06/2021.
```